walkthroughs/basic/build-flash-monitor.md

```diff
@@ -0,0 +1,36 @@
+# Building, Flashing, and Monitoring Your ESP-IDF Project
+
+### Building Your Project
+  Click the "Build Project" icon from the status barn !["Build Project"](../../media/walkthrough/icons/build.png) or use command `ESP-IDF: Build your Project`
+
+### Flashing to Device
+Before flashing, make sure your device is connected and you've selected the correct port
+  Click "Flash Device" icon from the status bar !["Flash Device"](../../media/walkthrough/icons/flash.png) or use the command `ESP-IDF: Flash (UART) your Project`
```

Comment:
I would suggest to list the general `ESP-IDF: Flash your project` instead of flash type specific command.